lib/wallet/src/Cardano/Wallet/Write/Tx/Redeemers.hs

```diff
@@ -0,0 +1,237 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{- HLINT ignore "Use <$>" -}
+
+-- |
+-- Copyright: © 2023 IOHK
+-- License: Apache-2.0
+--
+-- Module containing 'assignRedeemers'
```

Comment:
Hmm. Is there actually a function called `assignRedeemers`? 😆 

How about:
```suggestion
-- Support for script redeemers.
```

Reply:
👍 But going with `assignScriptRedeemers` — I find the alternative vague.